src/site/content/en/blog/receive-shared-data/index.md

```diff
@@ -0,0 +1,262 @@
+
+<figure class="w-figure w-figure--inline-right">
+  <img src="./wst-send.png" style="max-width: 400px;" alt="Android phone with the 'Share via' drawer open."/>
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    System-level share target picker with an installed PWA as an option.
+  </figcaption>
+</figure>
+
+## See Web Share Target in action
+
+1. Using Chrome 76 or later, open the [Web Share Target demo][demo].
+2. When prompted, click **Install** to add the app to your home screen, or
+   use the Chrome menu to add it to your home screen.
+3. Open any app that includes a native share intent, or use the Share button
+   in the demo app.
+4. From the target picker, choose **Web Share Test**.
+
+After sharing to the demo app, you should see all of the information sent via
+the web share target web app.
+
+## Register your app as a share target
+
+To register your app as a share target, it needs to meet [Chrome's
+installability criteria][installability]. In addition, before a user can share
+to your app, they must add it to their home screen. This prevents sites from
+randomly adding themselves to the user's share intent chooser and ensures that
+sharing is something that users want to do with your app.
+
+## Update your web app manifest
+
+To register your app as a share target, add a `share_target` entry to its [web
+app manifest][manifest]. This tells the operating system to include your app as
+an option in the intent chooser. What you add to the manifest controls the data
+that your app will accept. There are three common scenarios for the `share_target`
+entry:
+- Accepting basic information
+- Accepting application changes
+- Accepting files
+
+{% Aside %}
+You can only have one `share_target` per manifest, if you want to share to
+different places within your app, provide that as an option within the share
+target landing page.
+{% endAside %}
+
+### Accepting basic information
+
+If your target app is merely accepting basic information such as data, links,
+and text, add the following to the `manifest.json` file:
+
+```json
+"share_target": {
+  "action": "/share-target/",
+  "method": "GET",
+  "params": {
+    "title": "title",
+    "text": "text",
+    "url": "url"
+  }
+}
+```
+
+If your application already has a share URL scheme, you can replace the `param`
+values with your existing query parameters. For example, if your share URL
+scheme uses `body` instead of `text`, you could replace `"text": "text"` with `"text":
+"body"`.
+
+The `method` value defaults to `"GET"` if not provided. The `enctype` field, not
+shown in this example, indicates the [type of encoding][encoding] for the data.
+For the `"GET"` method, `enctype` defaults to `"application/x-www-form-urlencoded"` and
+is ignored if it's set to anything else.
+
+### Accepting application changes
+
+If the shared data changes the target app in some way—for example, saving a
+bookmark in the target application—set the `method` value to `"POST"` and include
```

Comment:
Used em dashes to more clearly demarcate the aside and switched to imperative mood, per GDDSG.